src/pages/app-development/extension-compatibility.md

```diff
@@ -0,0 +1,103 @@
+- &#8203;<Edition name="saas" /> SaaS supports a predefined list of events. You can manage events through the [Admin interface](../events/create-events.md) or [REST endpoints](../events/api.md).
+
+## Admin UI SDK
+
+- &#8203;<Edition name="paas" /> Use the Admin UI SDK version `3.0` or higher to enable IMS authentication tokens from [the shared context](../admin-ui-sdk/extension-points/index.md#shared-contexts). Refer to [troubleshooting](../admin-ui-sdk/troubleshooting.md#issues-upgrading-to-major-admin-ui-sdk-version) if you encounter version restrictions:
+
+    ```json
+    "magento/commerce-backend-sdk": "3.0.0 as 2.0.0"
+    ```
+
+- &#8203;<Edition name="saas" /> The Admin UI SDK and IMS modules are already configured. The [Admin interface](../admin-ui-sdk/configuration.md) works the same as it does in PaaS.
+- To enable Adobe authentication for actions called by the SPA, you can obtain IMS tokens that work in both PaaS and SaaS by using the following:
+
+    ```javascript
+    if (props.ims?.token) {
+        // When running inside Experience Cloud Shell, IMS token and orgId can be accessed via props.ims.
+        setImsToken(props.ims.token);
+        setImsOrgId(props.ims.org);
+    } else {
+        // Commerce PaaS & SaaS retrieves IMS token via sharedContext from Admin UI SDK v3.0+
+        // See https://developer.adobe.com/commerce/extensibility/admin-ui-sdk/extension-points/#shared-contexts
+        const guestConnection = await attach({ id: extensionId });
+        const context = guestConnection?.sharedContext;
+        setImsToken(context?.get('imsToken'));
+        setImsOrgId(context?.get('imsOrgId'));
+    }
+    ```
+
+    For more information, see the [full example](https://github.com/adobe/commerce-checkout-starter-kit/blob/main/commerce-backend-ui-1/web-src/src/components/MainPage.js).
+
+## Out-of-process extensibility modules
+
+- &#8203;<Edition name="paas" /> Requires composer installation.
+- &#8203;<Edition name="saas" /> Modules are pre-installed.
+- APIs function the same way in both environments:
+  - [custom attributes](https://developer.adobe.com/commerce/services/cloud/guides/custom-attributes/)
+  - [payment](../starter-kit/checkout/payment-reference.md)
+  - [shipping](../starter-kit/checkout/shipping-reference.md)
+  - [tax](../starter-kit/checkout/tax-reference.md)
+
+## Storefront integration and testing
+
+- &#8203;<Edition name="paas" /> [EDS Storefronts](https://experienceleague.adobe.com/developer/commerce/storefront/get-started/) require additional configuration, such as the Catalog and Storefront Compatibility Package. PaaS also provides the [Luma Storefront](https://experienceleague.adobe.com/docs/commerce/frontend/guide/storefront/luma.html), which is not available in SaaS.
```

Comment:
I don't know what the Catalog and Storefront Compatibility Package is, but the Storefront Compatibility Package is here: https://experienceleague.adobe.com/developer/commerce/storefront/setup/configuration/storefront-compatibility/install/